.github/workflows/docker-build-and-push.yaml

```
@@ -0,0 +1,34 @@
+          credentials_json: ${{ secrets.GCP_SERVICE_ACCOUNT_KEY }}
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@v2
+        
+      - name: Configure Docker for Artifact Registry
+        run: gcloud auth configure-docker us-east1-docker.pkg.dev --quiet
+        
+      - name: Build Docker image
+        run: docker build -f Dockerfile -t us-east1-docker.pkg.dev/terraform-practice-455719/ai-interview-python-server/ai-interview-python-server:latest .
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)